### DISCUSSION TITLE
ajout de composite

### DISCUSSION THREAD

**Discussion by edouar1** [witsa/synapps#729] [Q&A]
bonjour je veux crée un composite qui récupérer  l'état d un participant par exemple éclairage et aussi la contrôler a distance .
Du coup pour procéder est ce que je doit crée un composite avec différents liaison   ? 
cordialement. 
 @sarahamarwit

**Comment by Raphwit** [comment]
Bonjour, pour cela il vous faut un fournisseur de variable en écriture qui pointe vers cette ressource, je vous conseille de lire la documentation à propos des acteurs fournisseurs de données : https://witsa.github.io/synapps/concepts/actor-types/redy-wos-variable-source.html

Une fois ce fournisseur créée, il faudra renseigner la donnée du fournisseur avec la valeur qui nous intéresse

Par exemple, si la donnée est un booléen, vous aurez  un acteur "Bouton interrupteur" qui aura sa valeur liée en écriture à la donnée de votre acteur fournisseur.
De ce fait, lorsque la valeur de l'acteur "Bouton interrupteur" va changer, la variable va également changer.



Il n'est pas nécessaire de créer un composite pour uniquement cet élément, cependant, si vous souhaitez utiliser le même groupes d'acteurs au sein de plusieurs scènes différentes, il est préférable de regrouper ces acteurs dans un composite avec un fournisseur relatif, un tutoriel a été réaliser pour illustrer ce mode de fonctionnement : https://witsa.github.io/synapps/tutorials/composite-wos-relative.html
